public/video-ui/src/services/WorkflowApi.js

```diff
@@ -43,7 +43,14 @@ export default class WorkflowApi {
   static _getTrackInWorkflowPayload({ video, status, section, scheduledLaunchDate }) {
     const prodOffice = getProductionOffice();
 
-    // const scheduledLaunchDate = VideoUtils.getScheduledLaunch(video);
+    const published = video.contentChangeDetails.published;
+    let publishedDate = null;
+    if(published) publishedDate = moment(published.date);
+
+    const lastModified = video.contentChangeDetails.lastModified;
+    let lastModifiedDate = null;
+    if(lastModified) lastModifiedDate = moment(lastModified.date);
```

Comment:
this could be a ternary to avoid the `let`

```js
const { contentChangeDetails } = video;

const publishDate = contentChangeDetails.publish
  ? moment(contentChangeDetails.publish.date)
  : null;

const lastModifiedDate = contentChangeDetails.lastModified
  ? moment(contentChangeDetails.lastModified.date)
  : null;
```

Reply:
Thanks! I was having a hard time trying to write this nicely.